### PR TITLE
Update German translations

### PR DESCRIPTION
**Pull request by bitnapper**
- updated messages.pot
- updated messages.po for DE and added translations
- changed some occources of term master to primary

Some German translations could be not ideal and I couldnt figure out inclusive scentences in every translation but I'm happe to fix it later. Please let me know if your find some non ideal translations.

Makes PR [#5902](https://github.com/internetarchive/openlibrary/pull/5902) obsolete.

### CODE REVIEW

File: openlibrary/i18n/de/messages.po [hunk 1]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+#: lib/header_dropdown.html:46 lib/nav_head.html:107 login.html:10 login.html:72
+msgid "Log In"
+msgstr "Anmelden"
+
+#: login.html:15
+msgid ""
+"Please enter your <a href=\"https://archive.org\">Internet Archive</a> email and "
+"password to access your Open Library account."
+msgstr ""
+"Bitte geben Sie Ihr <a href=„https://archive.org“>Internet Archive</a> E-Mail-"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:253]:
```suggestion
"Bitte geben Sie Ihr <a href=\"https://archive.org\">Internet Archive</a> E-Mail-"
```
Inside of HTML tags, replace opening and closing double quote symbols with the straight double quote symbol

File: openlibrary/i18n/de/messages.po [hunk 2]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+msgstr "Wie können wir behilflich sein?"
+
+#: support.html:16
+msgid ""
+"Your question has been sent to our support team. We'll get back to you shortly. "
+"You can also <a href=\"https://twitter.com/openlibrary\">contact us on Twitter</"
+"a>."
+msgstr ""
+"Ihre Frage wurde an das Support-Team gesendet. Wir melden uns in Kürze. Sie "
+"können uns auch auf <a href=„https://twitter.com/openlibrary“>Twitter</a> "
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:516]:
```suggestion
"können uns auch auf <a href=\"https://twitter.com/openlibrary\">Twitter</a> "
```

File: openlibrary/i18n/de/messages.po [hunk 3]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+"kontaktieren."
+
+#: support.html:19
+msgid ""
+"Please check our <a href=\"/help/\">Help Pages</a> and <a href=\"/help/faq"
+"\">Frequently Asked Questions</a> (FAQ) to see if your question is answered "
+"there. Thank you."
+msgstr ""
+"Bitte sehen Sie auf unseren <a href=/help/>Hilfe Seiten</a> und den <a href=„/"
+"help/faq“>Frequently Asked Questions</a> (FAQ) to See if your question is "
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:526]:
```suggestion
"Bitte sehen Sie auf unseren <a href=\"/help/\">Hilfe Seiten</a> und den <a href=\"/"
"help/faq\">Frequently Asked Questions</a> (FAQ) to See if your question is "
```

File: openlibrary/i18n/de/messages.po [hunk 4]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+msgid "Incorrect. Please try again."
+msgstr "Fehler. Bitte erneut versuchen."
+
+#: account/create.html:81
+msgid ""
+"By signing up, you agree to the Internet Archive's <a href='//archive.org/about/"
+"terms.php' target='_blank'>Terms of Service</a>."
+msgstr ""
+"Die Nutzung der Open Library fällt unter die <a href=\"//archive.org/about/terms."
+"php\">Nutzungsbedingungen (AGB)</a> des Internet Archives."
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:995]:
```suggestion
"Die Nutzung der Open Library fällt unter die <a href='//archive.org/about/terms."
"php' target='_blank'>Nutzungsbedingungen (AGB)</a> des Internet Archives."
```

File: openlibrary/i18n/de/messages.po [hunk 5]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+
+# | msgid ""
+# | "Open Library is an initiative of the <a href=\"//archive.org/\">Internet
+# "
+# | "Archive</a>, a 501(c)(3) non-profit, building a digital library of "
+# | "Internet sites and other cultural artifacts in digital form.<br/>Other <a
+# "
+# | "href=\"//archive.org/projects/\">projects</a> include the <a href=\"//"
+# | "archive.org/web/\">Wayback Machine</a>, <a href=\"//archive.org/"
+# | "\">archive.org</a> and <a href=\"//archive-it.org\">archive-it.org</a>"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:3231]:
```suggestion
```

File: openlibrary/i18n/de/messages.po [hunk 6]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+"sites and other cultural artifacts in  digital form. Other <a href=\"//archive."
+"org/projects/\">projects</a> include the <a href=\"//archive.org/web/\">Wayback "
+"Machine</a>, <a href=\"//archive.org/\">archive.org</a> and <a href=\"//archive-"
+"it.org\">archive-it.org</a>"
 msgstr ""
-"Geben Sie ihr aktuelles Passwort ein (um ihre Identität zu prüfen) und wählen Sie ein neues Passwort um das alte zu ersetzen."
+"Open Library ist ein Projekt des <a href=\"//archive.org/\">Internet Archive</"
+"a>s, einer gemeinnützigen Organisation aus den USA nach 501(c)(3), die eine "
+"digitale Bibliothek der Internetseiten und anderer kultureller Gegenstände "
+"zusammenstellt.<br/>Andere <a href=\"//archive.org/projects/\">Projekte</a> sind "
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:3244]:
```suggestion
"zusammenstellt. Andere <a href=\"//archive.org/projects/\">Projekte</a> sind "
```

File: openlibrary/i18n/de/messages.po [hunk 7]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
-#: password.html:41
-msgid "Change"
-msgstr "Ändern"
+#: lib/nav_foot.html:78
+#, python-format
+msgid ""
+"version <a href=\"https://github.com/internetarchive/openlibrary/commit/%s\">%s</"
+"a>"
+msgstr ""
+"Version <a href=„https://github.com/internetarchive/openlibrary/commit/%s“>%s</a>"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:3254]:
```suggestion
"Version <a href=\"https://github.com/internetarchive/openlibrary/commit/%s\">%s</a>"
```

File: openlibrary/i18n/de/messages.po [hunk 8]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
-"password. Please check your inbox for a note from us."
+"<strong>You are not <a href=\"/account/login\" title=\"Log in now\">logged in</"
+"a>.</strong> Open Library will record your IP address and include it in this "
+"page's publicly accessible edit history. If you <a href=\"/account/create\" "
+"title=\"Create an Open Library account\">create an account</a>, your IP address "
+"is concealed and you can more easily keep track of all your edits and additions."
 msgstr ""
-"Wir haben Ihnen eine Nachricht mit Ihrem Benutzernamen und einer Anleitung zum Zurücksetzen Ihres Open-Library-Passworts an "
-"%(email)s geschickt. Bitte schauen Sie in Ihren Posteingang."
+"<strong>Sie sind nicht <a href=„/account/login“ title=„Anmelden“>eingeloggt</a>."
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:3337]:
```suggestion
"<strong>Sie sind nicht <a href=\"/account/login\" title=\"Anmelden\">eingeloggt</a>."
```

File: openlibrary/i18n/de/messages.po [hunk 9]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+"</strong> Die OpenLibrary zeichnet Ihre IP-Adresse auf und zeigt diese in der "
+"öffentlichen Änderungshistorie an. Wenn Sie einen <a href=„/account/create“ "
+"title=„Bei Open Library registrieren“>Account anlegen</a> möchten, wird Ihre IP-"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:3340]:
```suggestion
"öffentlichen Änderungshistorie an. Wenn Sie einen <a href=\"/account/create\" "
"title=\"Bei Open Library registrieren\">Account anlegen</a> möchten, wird Ihre IP-"
```

File: openlibrary/i18n/de/messages.po [hunk 10]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+msgstr "Anmelden"
 
-#: add.html:42 edit/edition.html:93
-msgid "Who is the publisher?"
-msgstr "Welcher Verlag?"
+#: lists/header.html:11 lists/header.html:24 lists/header.html:38
+#: lists/header.html:47 lists/header.html:56
+#, python-format
+msgid "<a href=\"%(href)s\">%(name)s</a> / Lists"
+msgstr "<a href=„%(href)s“>%(name)s</a> / Listen"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:3408]:
```suggestion
msgstr "<a href=\"%(href)s\">%(name)s</a> / Listen"
```

File: openlibrary/i18n/de/messages.po [hunk 11]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
 
-#: edit.html:75
-msgid "<p class=\"alert thanks\">Thank you for adding that book! Any more information you could provide would be wonderful!</p>"
-msgstr ""
-"<p class=\"alert thanks\">Vielen Dank für das Hinzufügen dieses Buches! Wenn es mehr Informationen gibt, die Sie über "
-"bereitstellen können, wäre das Wundervoll!</p>"
+#: lists/home.html:48
+#, python-format
+msgid "from <a href=\"%(key)s\">%(owner)s</a>"
+msgstr "von <a href=„%(key)s“>%(owner)s</a>"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:3487]:
```suggestion
msgstr "von <a href=\"%(key)s\">%(owner)s</a>"
```

File: openlibrary/i18n/de/messages.po [hunk 12]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+"möchten?"
+
+#: lists/lists.html:121
+msgid "No lists yet!"
+msgstr "Bisher keine Listen!"
 
-#: edition-show.html:80
+#: lists/preview.html:17
+msgid "by <a href=\"$owner.key\">You</a>"
+msgstr "von <a href=„$owner.key“>Ihnen</a>"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:3556]:
```suggestion
msgstr "von <a href=\"$owner.key\">Ihnen</a>"
```

File: openlibrary/i18n/de/messages.po [hunk 13]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
-#: edit/excerpts.html:86
-msgid "From page"
-msgstr "Von Seite"
+#: type/author/view.html:119
+#, python-format
+msgid ""
+"Showing all works by author. Would you like to see <a href=\"%(url)s\">only "
+"ebooks</a>?"
+msgstr ""
+"Zeigt all Werke eines Autors. Möchten Sie nur <a href=„%(url)s“>eBooks</a> sehen?"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:4241]:
```suggestion
"Zeigt all Werke eines Autors. Möchten Sie nur <a href=\"%(url)s\">eBooks</a> sehen?"
```

File: openlibrary/i18n/de/messages.po [hunk 14]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
-#: edit/excerpts.html:114
-msgid "Please provide an excerpt."
-msgstr "Bitte geben Sie einen Auszug an."
+#: type/author/view.html:121
+#, python-format
+msgid ""
+"Showing ebooks only. Would you like to see <a href=\"%(url)s\">everything</a> by "
+"this author?"
+msgstr ""
+"Zeige nur eBooks des Autors. Möchten Sie <a href=„%(url)s“>alles</a> von diesem "
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:4249]:
```suggestion
"Zeige nur eBooks des Autors. Möchten Sie <a href=\"%(url)s\">alles</a> von diesem "
```

File: openlibrary/i18n/de/messages.po [hunk 15]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
-msgid "More"
-msgstr "Mehr"
+#: type/edition/view.html:230 type/work/view.html:230
+#, python-format
+msgid ""
+"This edition doesn't have a description yet. Can you <b><a href='%(url)s'>add "
+"one</a></b>?"
+msgstr ""
+"Diese Ausgabe hat bisher keine Beschreibung. Können Sie eine <b><a "
+"href=‚%(url)s‘>hinzufügen</a></b>?"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:4374]:
```suggestion
"href='%(url)s'>hinzufügen</a></b>?"
```

File: openlibrary/i18n/de/messages.po [hunk 16]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+msgstr "Hinzugefügt von %(authorlink)s."
+
+# | msgid "an anonymous user"
+#: type/edition/view.html:313 type/work/view.html:313
+msgid "added anonymously."
+msgstr "anonym hinzugefügt."
+
+#: type/edition/view.html:321 type/work/view.html:321
+msgid "Links <span class=\"gray small sansserif\">outside Open Library</span>"
+msgstr "Links außerhalb der <span class=„gray small sansserif“>Open Library</span>"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:4417]:
```suggestion
msgstr "Links außerhalb der <span class=\"gray small sansserif\">Open Library</span>"
```

File: openlibrary/i18n/de/messages.po [hunk 17]
```diff
@@ -7,2187 +7,5520 @@ msgid ""
...
+msgid "Subscribe"
+msgstr "Abonnieren"
 
-#: nav_head.html:14 nav_head.html:91
-msgid "Random Book"
-msgstr "Zufälliges Buch"
+#: type/list/exports.html:36 type/list/exports.html:56
+#, python-format
+msgid "Watch activity via <a rel=\"nofollow\" href=\"%s\">Atom feed</a>"
+msgstr "Aktivitäten via <a rel=„nofollow“ href=„%s“>Atom feed</a> verfolgen"
```
Comment (jimchamp) [inline openlibrary/i18n/de/messages.po:4570]:
```suggestion
msgstr "Aktivitäten via <a rel=\"nofollow\" href=\"%s\">Atom feed</a> verfolgen"
```

**Comment by jimchamp** [comment]
Thanks for the updated translations!  I made some small tweaks to some of the strings containing HTML tags in order for the tests to pass.

**Comment by bitnapper** [comment]
@jimchamp Thanks, just fixed it in my version too. Any suggestions on a better po-file-editor? Pedit does not ignore the charactersubstition as every other code-editor does. I have turn it off systemwide but then I tend to forget turning it on and off again. I tried gtranslator but this one does not run very smoothly on mac too.

**Pull request merged**